src/api-controller.js

```diff
@@ -0,0 +1,41 @@
+const getMovies = () => fetch('https://api.tvmaze.com/shows')
+  .then((response) => response.json());
+
+const getlikes = () => fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/izzKZgflb43lj8vOIg9b/likes')
+  .then((response) => response.json());
+
+const getComments = (id) => fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/izzKZgflb43lj8vOIg9b/comments?item_id=${id}`)
+  .then((response) => {
+    if (response.ok) {
+      return response.json();
+    }
+
+    return [];
+  });
+
+const postLike = (id) => fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/izzKZgflb43lj8vOIg9b/likes', {
+  method: 'POST',
+  body: JSON.stringify({
+    item_id: id,
+  }),
```

Comment:
- [ ] I see you did a great job, but you should have used the `async and await` instead.

![ooooo](https://user-images.githubusercontent.com/61562626/142405254-331f029d-ff08-4983-b956-d0bbf1667f93.jpeg)